### DISCUSSION TITLE
KeyPair values aren’t appearing in the dropdown on the Specify stack details page, even though I have two keys in the console. See attached. Can there be any reason why?

### DISCUSSION THREAD

**Discussion by ybinstock** [mozilla/hubs#3258] [Hubs Cloud Questions] [answered]
See attached for my keys from my console, but they are not showing up in the dropdown. 

<img width="1162" alt="Screen Shot 2020-10-26 at 4 14 55 PM" src="https://user-images.githubusercontent.com/8645066/97781609-f1bd0480-1b8c-11eb-8444-09d14c358f59.png">

Is there something I'm missing?

**Comment by robinkwilson** [comment, marked as answer]
Your SSH key pairs are likely not in the same region as your cloud deploy. Both have to be the same. 

See this video clip to find which region you have selected when creating the keypair: https://youtu.be/2K7P8jFyHNc?t=188
